client/src/components/profile/Profile.js

```diff
@@ -125,6 +130,12 @@ return <React.Fragment>
                             <i className="glyphicon glyphicon-globe"></i>Sex : {this.state.user.gender}
                             <br />
                             <i className="glyphicon glyphicon-gift"></i>{this.state.user.phone_number}</p>
+                            <br />
+                            <span>Selfie Uploaded </span>{this.state.user.is_selfie_image_uploaded? <MDBIcon icon="check-circle" className="green-text pr-3"/> :<MDBIcon icon="times-circle" className="red-text pr-3"/> } 
+                            <br />
+                            <span>3D Avatar Generated </span>{this.state.user.is_selfie_model_uploaded? <React.Fragment><MDBIcon icon="check-circle" className="green-text pr-3"/> <br /> <a href={this.state.user.avatar_url} className="btn btn-primary">Download Avatar</a> </React.Fragment> 
```

Comment:
please check this. it is referring to avatar 3d and I am not sure if will mess up what the other folks added